package.json

```diff
@@ -70,7 +71,13 @@
     "0": "JSON doesn't support comments so I'm adding mine here.",
     "1": "==============================",
     "2": "Vue version is overridden because of the issue described in the 'tip' in",
-    "3": "the installation section of https://pinia.vuejs.org/ssr/nuxt.html"
+    "3": "the installation section of https://pinia.vuejs.org/ssr/nuxt.html",
+    "4": "==============================",
+    "5": "@rollup/rollup-linux-x64-gnu is an optional dependency as a fix for the issue that Rollup",
+    "6": "describes here: https://github.com/rollup/rollup/blob/f83b3151e93253a45f5b8ccb9ccb2e04214bc490/native.js#L59",
+    "7": "and which occurred for us when doing an installation with npm on Docker on CI. Their suggested fix",
+    "8": "does not work for our use case, because removing package-lock.json prevents the use of `npm ci`, so instead",
+    "9": "we use the solution suggested here: https://github.com/vitejs/vite/discussions/15532#discussioncomment-10192839"
```

Comment:
Tbh, I find this trick for putting comments into package.json pretty unreadable. Given that the dependencies etc being referred to aren't actually in the vicinity of the relevant section of package.json anyway, I'm not sure there's much benefit in keeping them here. How about moving them to a section in the README file dedicated to npm dependency notes? That way you could make all these urls into working links!

Reply:
That sounds better - they're pretty fiddly to write as well because of the line numbers!